docs/reference/webapp/switch_workspaces.md

```diff
@@ -2,8 +2,10 @@
 
 This feature turns out very convenient when users need to use different workspaces for different purposes.
 
-The **switch workspace** menu is located in the upper right-hand corner, where the user icon is shown. It displays the different workspaces available for the user.
+The **switch workspace** menu is located in the upper right-hand corner, where the user icon is shown. It displays a **log out** and a **view docs** button.
 
-In order to switch workspaces, users must click on the user icon and select another one. This will drive the user to the main page of the **Workspace**.
+The **log out** button allows users to log out or to change the workspace. After clicking on the user icon and choosing this option, users will see the main page, where they can log in again.
```

Comment:
I think you cannot really change the workspace now. I would simply remove this part.